pkg/plugin/manifest/serializer.go

```diff
@@ -0,0 +1,99 @@
+package manifest
+
```

Comment:
gooooooooooooooooo (as in goo, not go with a long o)

Reply:
love too write Java in Go